Makefile

```diff
@@ -79,7 +79,7 @@ touch-protos:
 	for proto in $(PROTO_GOS); do [ -f "./$${proto}" ] && touch "$${proto}" && echo "touched $${proto}"; done
 
 %.pb.go: $(PROTO_DEFS)
-# We use loki-build-image here which expects /src/loki so we bind mount the agent
+# We use loki-build2-image here which expects /src/loki so we bind mount the agent
```

Comment:
Ditto, also this reads about the relation to Loki which I'm not sure is intended.